app/Foliage/Meta.hs

```diff
@@ -214,7 +214,12 @@ revisionMetaCodec =
     .= revisionNumber
 
 timeCodec :: Toml.Key -> TomlCodec UTCTime
-timeCodec key = Toml.dimap (utcToZonedTime utc) zonedTimeToUTC $ Toml.zonedTime key
+timeCodec = Toml.textBy showTime parseTime
+  where
+    showTime = T.pack . formatTime defaultTimeLocale "%FT%T%QZ"
+    parseTime t = case parseTimeM True defaultTimeLocale "%FT%T%QZ" (T.unpack $ T.dropAround (`elem` ['\'', '\"']) t) of
```

Comment:
This doesn't seem to work … I do not understand yet exactly why :/

Reply:
That's because `textBy` will match text values and convert them as you indicate but in TOML `2022-08-22T10:38:45Z` is not text but a timestamp, so `textBy` fails to match.